src/Entities/Player.h

```diff
@@ -138,6 +138,15 @@ class cPlayer :
 
 	// tolua_begin
 
+	/** Prevent the palyer from moving and lock him into a_Location. */
+	void Freeze(const Vector3d & a_Location);
```

Comment:
Will this be enough for plugin export? I just added  it inside the `tolua_begin` are. I never exported commands before.